scripts/build_wasm.sh

```diff
@@ -46,18 +86,20 @@ command() {
     esac
 }
 
-case $1 in
-    "")
-        command "libs"
-        command "samples"
-        ;;
-    "libs")
-        command "libs"
-        ;;
-    "samples")
-        command "samples"
-        ;;
-    *)
-        echo "error: arg must be 'libs', 'samples', or empty to build both"
-        exit 1
```

Comment:
Let's reject invalid args so that we don't type e.g. `--releas` `lib` and get unintended results

Reply:
Changed to `--profile={deploy, profiling}`